packages/block-editor/src/components/writing-flow/use-select-all.js

```diff
@@ -31,6 +31,12 @@ export default function useSelectAll() {
 				return;
 			}
 
+			if (
+				event.target.classList.contains( 'editor-post-title__input' )
```

Comment:
I don't think code in the block editor package should use a classname from the editor package. The dependency is the other way, that editor package depends on block editor. 

But it also doesn't cause any issues, this will just be ignored in other editors.

It could be good to see if there are other options for solving this that don't rely on an editor class name, though.

Reply:
Conditions such as `isEmpty( getSelectionStart() ) && isEmpty( getSelectionEnd() ) ` will cause this test to fail.

https://github.com/WordPress/gutenberg/blob/v11.1.0/packages/e2e-tests/specs/editor/various/multi-block-selection.test.js#L656

Any good ideas to get the state that the cursor is on the title?

Reply:
It looks like @ellatrix iterated on this in #33699, sorry I couldn't provide any better suggestions.